website/docs/how-to/how-to-create-api-tokens.mdx

```diff
@@ -2,36 +2,9 @@
 title: How to create API Tokens
 ---
 
-import newTokenImage from '/img/admin_create_token_form.png';
+Depending on your [permissions](#api-token-permissions), you can create API tokens in the Unleash Admin UI in three ways:
```

Comment:
I will eventually remove this file, as I explain the same information on the API tokens docs page. But first I have to update all SDK readmes because they reference this file and break the link checker